README.md

```diff
@@ -113,6 +114,43 @@ job.getQueryResults().on('data', function(row) {});
 ```
 
 
+## Google Compute Engine
+
+- [API Documentation][gcloud-compute-docs]
+- [Official Documentation][cloud-compute-docs]
```

Comment:
Might want to watch out for this https://github.com/GoogleCloudPlatform/gcloud-node/pull/790